addons/account/models/account_bank_statement.py

```diff
@@ -505,6 +505,7 @@ class AccountBankStatementLine(models.Model):
     # == Business fields ==
     move_id = fields.Many2one(
         comodel_name='account.move',
+        auto_join=True,
```

Comment:
Beware: this bypasses access rights checks.
I guess that having access to bank statements implies having access to account moves.
If my guess is right, then this patch is safe.

Reply:
It is right indeed